packages/angular-workspace/example-client-app/src/app/app.module.ts

```diff
@@ -31,6 +31,7 @@ import { NimbleTableColumnMenuButtonModule } from '@ni/nimble-angular/table-colu
 import { NimbleRichTextViewerModule } from '@ni/nimble-angular/rich-text/viewer';
 import { NimbleRichTextEditorModule } from '@ni/nimble-angular/rich-text/editor';
 import { NimbleRichTextMentionUsersModule } from '@ni/nimble-angular/rich-text-mention/users';
+import { SprightChatModule } from '@ni/spright-angular/chat';
```

Comment:
We should be making an entry point per component and align with the component layout so imports look like:
```
import { SprightChatMessageModule } from '@ni/spright-angular/chat/message';
import { SprightChatConversationModule } from '@ni/spright-angular/chat/conversation';
```